FND/Places/Addresses.rdf

```diff
@@ -138,30 +139,32 @@
 	<owl:Class rdf:about="&fibo-fnd-plc-adr;ConventionalStreetAddress">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-plc-adr;PhysicalAddress"/>
 		<rdfs:subClassOf>
-			<owl:Class>
-				<owl:unionOf rdf:parseType="Collection">
-					<owl:Restriction>
-						<owl:onProperty rdf:resource="&fibo-fnd-plc-adr;hasAddressLine1"/>
-						<owl:maxQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:maxQualifiedCardinality>
-						<owl:onDataRange rdf:resource="&rdfs;Literal"/>
-					</owl:Restriction>
-					<owl:Restriction>
-						<owl:onProperty rdf:resource="&fibo-fnd-plc-adr;hasAddressLine2"/>
-						<owl:maxQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:maxQualifiedCardinality>
-						<owl:onDataRange rdf:resource="&rdfs;Literal"/>
-					</owl:Restriction>
-					<owl:Restriction>
-						<owl:onProperty rdf:resource="&fibo-fnd-plc-adr;hasAddressLine3"/>
-						<owl:maxQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:maxQualifiedCardinality>
-						<owl:onDataRange rdf:resource="&rdfs;Literal"/>
-					</owl:Restriction>
-					<owl:Restriction>
-						<owl:onProperty rdf:resource="&fibo-fnd-plc-adr;hasStreetAddress"/>
-						<owl:onClass rdf:resource="&fibo-fnd-plc-adr;StreetAddress"/>
-						<owl:maxQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:maxQualifiedCardinality>
-					</owl:Restriction>
-				</owl:unionOf>
-			</owl:Class>
+			<owl:Restriction>
```

Comment:
overall this does not require an address to have any address lines, nor a "street address". Was that the intent? 
Unlike the previous version it also allows more than 1 hasAddressLine2 which seems wrong. Why not set maxCard to 1?

Reply:
Ok - I've put the max cardinality restrictions back.  The difference in reasoning time is quite significant, but as long as folks run reasoners on machines that have sufficient memory, it is ok.  FIBO requires at least 32 GB memory to run a reasoner now unfortunately, though I'm investigating newer reasoners to see whether or not there are other options people can try.  HermiT blows up on FIBO prod on most common machines now, and this was an attempt to eliminate a costly restriction.

We should think about whether we need to move costly restrictions into a separate ontology so that people can run reasoners for testing without some of the more expensive constraints that we think are appropriate.

Reply:
Or move these constraint-like restrictions to SHACL, since AFAIK they will not usually reveal inconsistencies in the ontology.